base/logging_context_test.go

```diff
@@ -24,16 +23,19 @@ const standardMessage = "foobar"
 func requireLogIs(t testing.TB, s string, f func()) {
 	b := bytes.Buffer{}
 
+	config := &ConsoleLoggerConfig{
+		ColorEnabled: BoolPtr(false),
+	}
+	tempLogger, err := NewConsoleLogger(TestCtx(t), false, config)
+	require.NoError(t, err)
+	tempLogger.logger.SetOutput(&b)
 	timestampLength := len(time.Now().Format(ISO8601Format) + " ")
 
-	logger := consoleLogger.Load()
 	// Temporarily override logger output for the given function call
-	originalColor := logger.ColorEnabled
-	logger.ColorEnabled = false
-	logger.logger.SetOutput(&b)
+	oldLogger := consoleLogger.Load()
+	consoleLogger.Load()
```

Comment:
Line 36 should be `Store(tempLogger)` not `Load`, but we can probably just use `Swap` here instead?

```suggestion
	oldLogger := consoleLogger.Swap(tempLogger)
```